### PR TITLE
Initial commit, Login page

### PR DESCRIPTION
**Pull request by RetroOosthuizen**
Created the login page with JS and CSS.
Redirect from the initial page to the loginComponent. 
@jaderabbit

### CODE REVIEW

**Comment by zschyff** [comment]
It looks fine.

File: src/LoginComponent.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+
+import React, { Component } from 'react';
+import './sass/style.sass';
+import './App.css';
+import './css/style.css';
+import loginJS from './js/login.js';
+import $ from 'jquery';
```
Comment (WesleyBuck) [inline src/LoginComponent.js:7]: why are you adding jquery to a react project?

File: src/LoginComponent.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
+
+import React, { Component } from 'react';
+import './sass/style.sass';
+import './App.css';
```
Comment (WesleyBuck) [inline src/LoginComponent.js:4]: app.css is already added to app.js??? It propagates to LoginComponent.js

**Review by WesleyBuck — CHANGES_REQUESTED** [review]

**Review by laurent-mbuyu — CHANGES_REQUESTED** [review]
Looking at the index.js file, I can see that it lands on the App component. You should include routing to the appropriate page so that it can land on the login when it's needed. Refer to my pull request(Settings) for more information.

File: src/js/login.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import $ from 'jquery';
```
Comment (grootendorste) [inline src/js/login.js:1]: It's very dangerous using jquery along with react. We already know you can do jquery, I believe the exercise is to do it in react.

File: src/js/login.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import $ from 'jquery';
+
+$(document).ready(function() {
+
+var placeholderText = $("label[for='username']").text();
```
Comment (grootendorste) [inline src/js/login.js:5]: In the future, use let instead of var, and const if it's a constant.

**Review by grootendorste — CHANGES_REQUESTED** [review]
Please remove dependency of jquery.

**Pull request closed without merging**